core/src/epicli/data/common/validation/configuration/grafana.yml

```diff
@@ -335,16 +323,15 @@ properties:
       provider:
         type: string
       key_file:
-        type: 'null'
+        type: string
       bucket:
-        type: 'null'
+        type: string'
```

Comment:
string' 🤔

Reply:
A test to check your alertness, you passed.